pylc3/unittest/lc3_unit_test_case_test.py

```diff
@@ -601,23 +602,30 @@ def testReplayString(self):
 
         blob = self.preconditions._formBlob()
 
-        self.assertEqual(blob,
-            '\x01\x01\x00\x00\x00'
-            '\x02\x01\x00\x00\x00'
-            '\x03\x01\x00\x00\x00'
-            '\x04\x00\x00\x00\x00'
-            '\x05\x00\x00\x00\x00'
-            '\x06\xff\xff\xff\xff'
-            '\x07\x00\x80\x00\x00'
-            '\x11\x01\x00\x00\x007\x01\x00\x00\x00\x01@'
-            '\x12\x00\x00\x00\x00\x01\x00\x00\x00\x00\x05'
-            '\x13\x03\x00\x00\x00AHH\x01\x00\x00\x00\x07\x00'
-            '\x14\x04\x00\x00\x00BLAH\x01\x00\x00\x00\xa6\x02'
-            '\x15\x06\x00\x00\x00CAWCAW\x05\x00\x00\x00\x05\x00\x02\x00\t\x00\x000\xff\xff'
-            '\x16\x04\x00\x00\x00PAPA\x04\x00\x00\x00M\x00A\x00M\x00A\x00'
-            '\x17\x00\x00\x00\x00\x06\x00\x00\x00R\x00A\x00H\x00R\x00A\x00H\x00'
-            '\x18\x04\x00\x00\x00TATA\x06\x00\x00\x00\x05\x00\x04@\x00\x80\x02\x00\x05\x00\x07\x00')
+        expected_blob = six.b('\x01\x01\x00\x00\x00' \
+                    '\x02\x01\x00\x00\x00' \
+                    '\x03\x01\x00\x00\x00' \
+                    '\x04\x00\x00\x00\x00' \
+                    '\x05\x00\x00\x00\x00' \
+                    '\x06\xff\xff\xff\xff' \
+                    '\x07\x00\x80\x00\x00' \
+                    '\x11\x01\x00\x00\x007\x01\x00\x00\x00\x01@' \
+                    '\x12\x00\x00\x00\x00\x01\x00\x00\x00\x00\x05' \
+                    '\x13\x03\x00\x00\x00AHH\x01\x00\x00\x00\x07\x00' \
+                    '\x14\x04\x00\x00\x00BLAH\x01\x00\x00\x00\xa6\x02' \
+                    '\x15\x06\x00\x00\x00CAWCAW\x05\x00\x00\x00\x05\x00\x02\x00\t\x00\x000\xff\xff' \
+                    '\x16\x04\x00\x00\x00PAPA\x04\x00\x00\x00M\x00A\x00M\x00A\x00' \
+                    '\x17\x00\x00\x00\x00\x06\x00\x00\x00R\x00A\x00H\x00R\x00A\x00H\x00' \
+                    '\x18\x04\x00\x00\x00TATA\x06\x00\x00\x00\x05\x00\x04@\x00\x80\x02\x00\x05\x00\x07\x00')
+
+        with open('expected.dat', 'wb') as f:
```

Comment:
What's with these 4 lines of code?

Reply:
oh I assume this was here for debugging as well, you did mention they didn't match up on your system.